### ISSUE TITLE
AT not reading issue source element once balloon is minimized

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#165]
Based on our partner's feedback:

> JAWS not reading the selected Element in editor body when we close the dialog. For ex: when the dialog shows error for some text should be marked as list and when we close the dialog, it selects all the text that should be structured as list but no info. Please see screenshots below.
